task_script_utils/datetime_parser/datetime_info.py

```diff
@@ -534,11 +496,7 @@ def long_date_format(self) -> str:
                 f"YYYY"
             )
         else:
-            date_fmt = (
-                f"{self.token_month} "
-                f"{self.token_day} "
-                f"YYYY"
-            )
+            date_fmt = f"{self.token_month} " f"{self.token_day} " f"YYYY"
```

Comment:
I'm guessing black auto-formatted this weird, but could you fix it to be more readable:
```suggestion
            date_fmt = f"{self.token_month} {self.token_day} YYYY"
```

Reply:
fixed